classes/invitation/invitations/userRoleAssignment/handlers/UserRoleAssignmentInviteUIController.php

```diff
@@ -103,9 +106,9 @@ public function createHandle(Request $request, $userId = null): void
             'id' => 'invitationWizard',
             'name' => __('invitation.wizard.pageTitle'),
         ];
-        $steps = new SendInvitationStep();
+        $steps = $this->invitation->buildSendSteps($context, $user);
```

Comment:
I would expect the `buildSendSteps` to be part of the handler more than of the `invitation` business objects